tests/utils/test_utils.py

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import os
+
+from living_documentation_generator.utils.utils import (
+    make_issue_key,
+    sanitize_filename,
+    make_absolute_path,
+    get_action_input,
+    set_action_output,
+    set_action_failed,
+)
+
+
+# make_issue_key
+
+
+def test_make_issue_key():
+    organization_name = "org"
+    repository_name = "repo"
+    issue_number = 123
+
+    expected_key = "org/repo/123"
+    actual_key = make_issue_key(organization_name, repository_name, issue_number)
+
+    assert actual_key == expected_key
+
+
+# sanitize_filename
+
+
+@pytest.mark.parametrize(
+    "filename_example, expected_filename",
+    [
+        ("in<>va::lid//.fi|le?na*me.txt", "invalid.filename.txt"),  # Remove invalid characters for Windows filenames
+        ("another..invalid...filename.txt", "another.invalid.filename.txt"),  # Reduce consecutive periods
+        (
+            "filename   with   spaces.txt",
+            "filename_with_spaces.txt",
+        ),  # Reduce consecutive spaces to a single space and replace spaces with '_'
+    ],
+)
+def test_sanitize_filename(filename_example, expected_filename):
+    actual_filename = sanitize_filename(filename_example)
+    assert actual_filename == expected_filename
+
+
+# make_absolute_path
+
+
+def test_make_absolute_path_from_relative_path():
```

Comment:
Here, we are testing 3rd party methods.

Reply:
You are right, removing this test from the file.